### ISSUE TITLE
card must be deleted when the user clicks outside the card containers.

### ISSUE THREAD

**Issue by ogerly** [Schamper/nodebb-plugin-cards#7]
first good work! I've discovered something.
if user image with the mouse over the then the card goes on. OK
when I picture user leave from the mouse again it will be deleted. OK
but when I click on the user then the card stops and deletes itself.
card must be deleted when the user clicks outside the card containers.
nodebb 1.0.x

**Comment by Schamper** [comment]
Could you please try and be more specific? Not really understanding what the issue is.

**Comment by JoshRickers** [comment]
Might be something similar.
Hover your mouse over a user name and the card opens, click on the username and the page changes to the  user profile but the card is still displayed. Cannot be destroyed unless you refresh.

http://i.imgur.com/SG4pTKP.png

**Comment by revir** [comment]
@ogerly 
@JoshRickers 
Hi guys, I fixed this on [nodebb-plugin-cards2](https://github.com/revir/nodebb-plugin-cards2), and used on [my website](https://v2mm.tech). check it :)

**Comment by Schamper** [comment]
@revir you didn't think about creating a PR?

**Comment by revir** [comment]
@Schamper 
Yeah, I just created a PR, but I changed the README and Package.json for using on my website, could you convert them back?
